src/api/app/views/webui2/webui/users/tasks/index.html.haml

```diff
@@ -0,0 +1,87 @@
+            class: 'nav-link text-nowrap', 'data-toggle': 'tab', 'aria-controls': 'requests-out-tab', 'aria-selected': 'false', role: 'tab' }
+            Outgoing Requests
+        %li.nav-item
+          %a#requests-declined-tab{ href: '#requests-declined', title: "Requests from #{User.current.login} that are declined",
+            class: 'nav-link text-nowrap', 'data-toggle': 'tab', 'aria-controls': 'requests-declined-tab', 'aria-selected': 'false', role: 'tab' }
+            Declined Requests (#{User.current.declined_requests.count})
+        %li.nav-item
+          %a#all-requests-tab{ href: '#all-requests', title: "All Requests from #{User.current.login}",
+            class: 'nav-link text-nowrap', 'data-toggle': 'tab', 'aria-controls': 'all-requests-tab', 'aria-selected': 'false', role: 'tab' }
+            All Requests
+        %li.nav-item.dropdown
+          %a.nav-link.dropdown-toggle{ href: '#', 'data-toggle': 'dropdown', 'role': 'button', 'aria-expanded': 'false', 'aria-haspopup': 'true' }
+          .dropdown-menu.dropdown-menu-right
+    .card-body
+      .tab-content
+        .tab-pane.fade.show.active#requests-in{ role: 'tabpanel', 'aria-labelledby': 'requests-in-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: user_requests_path(User.current) })
+        .tab-pane.fade#requests-out{ role: 'tabpanel', 'aria-labelledby': 'requests-out-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_out_table', source_url: user_requests_path(User.current) })
+        .tab-pane.fade#requests-declined{ role: 'tabpanel', 'aria-labelledby': 'requests-declined-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_declined_table', source_url: user_requests_path(User.current) })
+        .tab-pane.fade#all-requests{ role: 'tabpanel', 'aria-labelledby': 'all-requests-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'all_requests_table', source_url: user_requests_path(User.current) })
+
+  - if User.current.involved_patchinfos.present?
+    .card.mt-3#patchinfos
+      .bg-light
+        %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+          %li.nav-item
+            %a{ href: '#patchinfos-in', title: "Requests that #{User.current.login} has to merge", class: 'nav-link text-nowrap active' }
+              Maintenance Requests
+      .card-body
+        .tab-content#patchinfos-in
+          %table.responsive.table.table-striped.table-bordered#open-patchinfos-table{ title: 'Running Maintenance Updates', width: '100%' }
+            %thead
+              %tr
+                %th Project
+                %th Package
+                %th Issues
+                %th Actions
+            %tbody
+              - User.current.involved_patchinfos.each do |pi|
```

Comment:
you are doing this query `involved_patchinfos` twice. You could cache it in the model or assign it to a variable

Reply:
Give `pi` a better name :wink:

Reply:
like number314 to make it less obvious :)